corehq/apps/translations/integrations/transifex/parser.py

```diff
@@ -13,7 +13,7 @@
 
 CONTEXT_REGEXS = {
     # Module or Form: sheet name for module/form: unique id
-    'module_and_forms_sheet': r'^(Module|Form):(\w+):(\w+)$',
+    'module_and_forms_sheet': r'^(Menu|Form):(\w+):(\w+)$',
```

Comment:
should the other instances of module be changed to menu as well

Reply:
Are there particular ones you noticed? It's a bit hairy in this file because the transifex resources are, and should be, still named "module" - I don't want to be changing the names of anything in transifex. I'm fairly dependent on @mkangia to review these changes.